pkg/storage/tpr/install_types.go

```diff
@@ -100,8 +105,12 @@ func (i *Installer) InstallTypes() error {
 
 	if allErrMsg != "" {
 		glog.Errorf("Failed to create Third Party Resource:\n%s)", allErrMsg)
-		return fmt.Errorf("Failed to create Third Party Resource:\n%s)", allErrMsg)
+		return ErrTPRInstall{fmt.Sprintf("Failed to create Third Party Resource:\n%s)", allErrMsg)}
```

Comment:
@simonleung8 I'd leave the newline out. Practically speaking, the only time it matters is when we log it, and almost all of our logging functionality auto-appends a newline. If we need to list multiple errors, we'll put them into a slice of `error`s